pkgs/by-name/at/atuin-desktop/package.nix

```diff
@@ -0,0 +1,131 @@
+{ lib,
+  stdenv,
+  fetchurl,
+  dpkg,
+  wrapGAppsHook4,
+  dbus,
+  zlib,
+  fontconfig,
+  openssl,
+  libappindicator,
+  webkitgtk_4_1,
+  gtk3,
+  pango,
+  cairo,
+  gdk-pixbuf,
+  libsoup_3,
+  glib,
+  glib-networking,
+  gcc,
+  pkg-config,
+}:
+
+
+stdenv.mkDerivation rec {
```

Comment:
Lets remove this `rec` in favor of using `stdenv.mkDerivation (finalAttrs: { ... })`. This pattern allow the values to be properly updated when applying packages override